packages/mock-account-servicing-lib/src/setupFromSeed.ts

```diff
@@ -35,7 +43,9 @@ export async function setupFromSeed(config: Config): Promise<void> {
     )
 
     assets[code] = asset
-    console.log(JSON.stringify({ asset, initialLiquidity }, null, 2))
+    if (debug) {
+      console.log(JSON.stringify({ asset, initialLiquidity }, null, 2))
+    }
```

Comment:
the setup from seed and requests just log everything in our local environment but this made less sense for testing. This debug option was a kinda quick solution - perhaps a pino debug log would make more sense?

Reply:
We want to more towards proper logging so yes, please let's do pino debug logs.

Reply:
I changed to pino but these logs were basically unreadable (lots of fairly large objects) so I added [pino-pretty](https://www.npmjs.com/package/pino-pretty) which helps (looks similar to how it was). It's just really tedious to digest any of it without `pino-pretty` IMO.

Without pino-pretty:

<img width="892" alt="Screenshot 2024-03-16 at 9 49 28 PM" src="https://github.com/interledger/rafiki/assets/12960453/4c050e51-31a8-4c16-88e5-a8e79587b1f5">

With pino-pretty:
<img width="898" alt="Screenshot 2024-03-16 at 10 08 23 PM" src="https://github.com/interledger/rafiki/assets/12960453/2283d8a9-2c55-4988-9df6-8792a76da2dc">

For whatever it's worth, this is something we can do elsewhere (backend, auth, etc), if we wanted. It just probably shouldn't be used in production. If this wont be used in prod we can hardcode it on, but currently it's controlled by the `pinoPretty` option (false by default) because `mock-account-servicing-entity` actually runs with a `NODE_ENV` in production due to some expectations by remix, if I recall correctly, and I wasn't sure if reading the env in the library made total sense anyways.